client/src/css/app.sass

```diff
@@ -72,6 +72,13 @@ a[target="_blank"]::after
         .q-field__control
             height: auto
 
+th.text-right, th.text-left, th.text-center, .q-table tbody td, .q-table__bottom-item,
```

Comment:
Submissions page
![Screenshot of Submissions page showing 14px font size](https://github.com/MESH-Research/Pilcrow/assets/2668987/ee6cb930-c7f3-49c0-81cb-0b6bc93e9d0b)

Reply:
These now appear as 16px.